base_location_nuts/migrations/16.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+import logging
+
+from requests.exceptions import HTTPError
+
+from odoo import SUPERUSER_ID, api
+
+_logger = logging.getLogger(__name__)
+
+
+def migrate(cr, version):
+    # Create or update NUTs
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    try:
+        env["nuts.import"].import_update_partner_nuts()
```

Comment:
issue: AFAICS there are no changes related to our data model, right? Only to the way we import that from official sources.

Then, is it really necessary to force a NUTS update while upgrading?
- Not: remove migration script.
- Yes: you cannot load a model you're declaring in the pre-migration stage, only in post-migration.

Reply:
Well, the last RAMON file used in this module is from 2013. I came across several differences between the quantity of nuts loaded because UK leaving the EU for instance. This update action should be declared in the configuration steps instead?

Reply:
That would be better.

In any case, when RAMON was working, data could still be very outdated since 2013 (it's been 11 years). So it doesn't seem like a migration problem, but something that can be done later.